crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/function.py

```diff
@@ -231,3 +256,19 @@ def f42(
     c,
 ):
     pass
+
+
+# Type parameter comments
+
+def type_param_comments[
+    # leading comment
+    A,
+
+    # in between comment
+
+    B, 
+    # another leading comment
+    C,
+    D, # trailing comment
+]():
+    pass
```

Comment:
Would you mind adding an example with a trailing `[` and leading `]` comment


```suggestion
def type_param_comments[ # trailing bracket comment
    # leading comment
    A,

    # in between comment

    B, 
    # another leading comment
    C,
    D, # trailing comment
    # trailing end of line comment
]():
    pass
```